vtex-apps/my-account
# Event accountView

#### What did you change? \*

- MyAccount now emits aa accountView event.

#### Why? \*

- Needed for anti-fraud purposes.

#### How to test it? \*

Login to: https://cabaceiras--storecomponents.myvtex.com/account#/profile

On console.log see the: `window.pixelManagerEvent`

![image](https://user-images.githubusercontent.com/3926634/63863899-55461380-c985-11e9-8626-0d1bb24f9470.png)


#### Related to / Depends on?

https://app.clubhouse.io/vtex/story/18066/adicionar-eventos-no-my-account-para-ferramentas-de-antifraud

#### Types of changes \*
